src/map/utils/charutils.cpp

```diff
@@ -96,6 +96,12 @@ This file is part of DarkStar-server source code.
 #include "petutils.h"
 #include "zoneutils.h"
 
+enum ITEMCHECK
```

Comment:
probably put this in item_armor.h since it's directly related to SCRIPTTYPE

Reply:
also, it should be an enum class

Reply:
word.